datafusion/physical-plan/src/aggregates/topk/heap.rs

```diff
@@ -361,9 +385,9 @@ impl<VAL: ValueType> HeapItem<VAL> {
 impl<VAL: ValueType> Debug for HeapItem<VAL> {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         f.write_str("bucket=")?;
-        self.map_idx.fmt(f)?;
+        Debug::fmt(&self.map_idx, f)?;
```

Comment:
is this change needed?

Reply:
I think so. If it's not changed, the compiler will fail and report `fmt` being ambiguous

Reply:
Makes sense -- thank you